service/internal/access/v2/pdp.go

```diff
@@ -227,6 +245,111 @@ func (p *PolicyDecisionPoint) GetDecision(
 	return decision, nil
 }
 
+func (p *PolicyDecisionPoint) GetDecisionRegisteredResource(
```

Comment:
Maybe `GetDecisionRegisteredResourceAsEntity`? 🤷

Reply:
IMO I don't think it needs a different name since we don't call the original one `GetDecisionEntityRepresentationAsEntity`. Seems self-explanatory to me unless I'm missing something? Alternatively we could rename the `registeredResourceValueFQN` param to `entityRegisteredResourceValueFQN` to make it consistent with the `GetDecision` method.

Reply:
param has been renamed